pythonPackages.nix

```diff
@@ -7,4 +7,18 @@ in {
   pyscf = callPackage ./pyscf { };
   pyquante = callPackage ./pyquante { };
   pychemps2 = callPackage ./chemps2/PyChemMPS2.nix { };
+  qcelemental = callPackage ./qcelemental { };
+  qcengine = callPackage ./qcengine { };
+  gau2grid-1_3_1 = callPackage ./gau2grid { version = "1.3.1"; sha256 = "0zkfil7cxjip79wqvhljk1ifjq0cwxzx6wlxgp63b6wbagma0i12"; };
+  gau2grid-2_0_4 = callPackage ./gau2grid { version = "2.0.4"; sha256 = "0qypq8iax0n6yfi4223zya468v24b60nr0x43ypmsafj0104zqa6"; };
```

Comment:
```
unpacking source archive /build/v2.0.7.tar.gz
hash mismatch in fixed-output derivation '/nix/store/0s3ncb296vpdg8gni1c3wm3rff74f8bl-source':
  wanted: sha256:0qypq8iax0n6yfi4223zya468v24b60nr0x43ypmsafj0104zqa6
  got:    sha256:1c01flf0xlm68dsv0a4n6bxglyj3x5mj8shmxglfpywbf74i3vnb
```

Reply:
Always build version 2.0.7 due to stupid error (overriding the version argument from the function in the derivation) and hence the mismatches for other versions. Fixed by [https://github.com/markuskowa/NixOS-QChem/pull/6/commits/67d2f28b597671907ff57e51cfb280605c2bef35](https://github.com/markuskowa/NixOS-QChem/pull/6/commits/67d2f28b597671907ff57e51cfb280605c2bef35)